Balancer-Bots/Stable-Pool-Depegging/src/finding.ts

```diff
@@ -25,11 +26,12 @@ export function createDecreaseThresholdFinding(
   pool: string,
   startValue: ethers.BigNumber,
   endValue: ethers.BigNumber,
-  decrease: ethers.BigNumber
+  decrease: ethers.BigNumber,
+  poolName: string
 ): Finding {
   return Finding.from({
-    name: "High Stable Pool Amplification Parameter Decrease",
-    description: "A stable pool amplification parameter will have a significant decrease",
+    name: `High Stable Pool Amplification Parameter Decrease in ${poolName}`,
+    description: `${poolName}'s amplification parameter will have a decrease by ${decrease}`,
```

Comment:
not showing `endValue`?

Reply:
`endValue` can be added right 👍 

`will have` because of [the contract](https://etherscan.io/address/0xfeadd389a5c427952d8fdb8057d6c8ba1156cc56#code#F1#L33)
basically 

>  // This contract uses timestamps to slowly update its Amplification parameter over time. These changes must occur. 
> // over a minimum time period much larger than the blocktime, making timestamp manipulation a non-issue